liquidctl/cli.py

```diff
@@ -12,45 +12,46 @@
+  --release <number>                 Filter devices by hexadecimal release number
+  --serial <number>                  Filter devices by serial number
+  --bus <bus>                        Filter devices by bus
+  --address <address>                Filter devices by address in bus
+  --usb-port <port>                  Filter devices by USB port in bus
 
 Animation options (devices/modes can support zero or more):
-  --speed <value>                Abstract animation speed (device/mode specific)
-  --time-per-color <value>       Time to wait on each color (seconds)
-  --time-off <value>             Time to wait with the LED turned off (seconds)
-  --alert-threshold <number>     Threshold temperature for a visual alert (°C)
-  --alert-color <color>          Color used by the visual high temperature alert
-  --direction <string>           If the pattern should move forward or backward.
-  --start-led <number>           The first led to start the effect at
-  --maximum-leds <number>        The number of LED's the effect should apply to
+  --speed <value>                    Abstract animation speed (device/mode specific)
+  --time-per-color <value>           Time to wait on each color (seconds)
+  --time-off <value>                 Time to wait with the LED turned off (seconds)
+  --alert-threshold <number>         Threshold temperature for a visual alert (°C)
+  --alert-color <color>              Color used by the visual high temperature alert
+  --direction <string>               If the pattern should move forward or backward.
+  --start-led <number>               The first led to start the effect at
+  --maximum-leds <number>            The number of LED's the effect should apply to
 
 Other device options:
-  --single-12v-ocp               Enable single rail +12V OCP
-  --pump-mode <mode>             Set the pump mode (certain Corsair coolers)
-  --temperature-sensor <number>  The temperature sensor number for the Commander Pro
-  --legacy-690lc                 Use Asetek 690LC in legacy mode (old Krakens)
-  --non-volatile                 Store on non-volatile controller memory
-  --direct-access                Directly access the device despite kernel drivers
-  --unsafe <features>            Comma-separated bleeding-edge features to enable
+  --single-12v-ocp                   Enable single rail +12V OCP
+  --pump-mode <mode>                 Set the pump mode (certain Corsair coolers)
+  --temperature-sensor <number>      The temperature sensor number for the Commander Pro
+  --legacy-690lc                     Use Asetek 690LC in legacy mode (old Krakens)
+  --non-volatile                     Store on non-volatile controller memory
+  --direct-access                    Directly access the device despite kernel drivers
+  --unsafe <features>                Comma-separated bleeding-edge features to enable
+  --fan-mode <channel>:<mode>[,...]  The fan modes to configure when initializing the device
```

Comment:
```suggestion
  --fan-mode <channel>:<mode>[,...]  Set the mode for each fan (certain Corsair devices)
```

I also think it makes more sense to order this new option just after `--pump-mode`.

Reply:
fixed in 522976b